.github/workflows/wheel_macos_x86_64.yml

```
@@ -0,0 +1,210 @@
+name: Wheel::MacOS::Intel
+
+# **What it does**: Builds python wheels for MacOS (10.15) architecture x86_64 and store it as artifacts.
+#                   Python versions: 3.9, 3.10, 3.11.
+# **Why we have it**: To build wheels for pennylane-lightning installation.
+# **Who does it impact**: Wheels to be uploaded to PyPI.
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  release:
+    types: [published]
+
+env:
+  MACOSX_DEPLOYMENT_TARGET: 10.15
+
+concurrency:
+  group: wheel_macos_x86_64-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  set_wheel_build_matrix:
+    name: "Set wheel build matrix"
+    uses: ./.github/workflows/set_wheel_build_matrix.yml
+    with:
+      event_name: ${{ github.event_name }}
+
+  build_dependencies:
+    needs: [set_wheel_build_matrix]
+    strategy:
+      matrix:
+        os: [macos-12]
+        arch: [x86_64]
+        exec_model: ${{ fromJson(needs.set_wheel_build_matrix.outputs.exec_model) }}
+        kokkos_version: ${{ fromJson(needs.set_wheel_build_matrix.outputs.kokkos_version) }}
+
+    name: Kokkos core & kernels (${{ matrix.exec_model }}::${{ matrix.arch }} )
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Cache installation directories
+        id: kokkos-cache
+        uses: actions/cache@v3
+        with:
+          path: ${{ github.workspace}}/Kokkos_install/${{ matrix.exec_model }}
+          key: ${{ matrix.os }}-kokkos${{ matrix.kokkos_version }}-${{ matrix.exec_model }}
+
+      - name: Install clang
+        run: |
+          brew install libomp
+
+      - name: Clone Kokkos libs
+        if: steps.kokkos-cache.outputs.cache-hit != 'true'
+        run: |
+          git clone https://github.com/kokkos/kokkos.git
+          cd kokkos
+          git checkout ${{ matrix.kokkos_version }}
+          cd -
+          pushd . &> /dev/null
+          git clone https://github.com/kokkos/kokkos-kernels.git
+          cd kokkos-kernels
+          git checkout ${{ matrix.kokkos_version }}
+          cd -
+
+      - uses: actions/setup-python@v4
+        name: Install Python
+        with:
+          python-version: '3.9'
```

To make the style more uniform
```suggestion
          python-version: 3.9
```

---

The issue here is using 3.9 instead of '3.9' is that when you switch to 3.10, it doesn't register the 0 and can default to 3.1 ( we ran into this issue previously, not sure if it still applies, but safest to assume it does). It'd be better to be uniform everywhere with quotes I think.

---

Done!